### PR TITLE
refactor escrow cancel transactions

### PR DESCRIPTION
**Pull request by jonathanlei**
## High Level Overview of Change
PR split from https://github.com/ripple/explorer/pull/308

### Type of Change

- [X ] Refactor (non-breaking change that only restructures code)

### testing 
all tests passed

### CODE REVIEW

**Review by ckniffen — DISMISSED** [review]

File: src/containers/shared/components/Transaction/EscrowCancel/Simple.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import Account from '../../Account'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { SimpleRow } from '../SimpleRow'
+import { Amount } from '../../Amount'
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/EscrowCancel/Simple.tsx:7]: You should delete `src/containers/Transactions/test/mock_data/EscrowCancel.json`

File: src/containers/shared/components/Transaction/EscrowCancel/TableDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  const { instructions } = props
+  const { owner, sequence, ticketSequence } = instructions
+  return (
+    <div className="escrow">
+      {owner && (
+        <div>
+          <span className="label">{t('cancel_escrow')}</span>
+          <span className="account">{owner}</span>
+          <span>
+            -{sequence !== 0 ? sequence : `${ticketSequence} (Ticket)`}
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/EscrowCancel/TableDetail.tsx:15]: This needs i18n.

File: src/containers/shared/components/Transaction/types.ts [hunk 3]
```diff
@@ -57,6 +57,7 @@ export type TransactionTableDetailComponent =
 export interface TransactionDescriptionProps<T = any> {
   data: {
     tx: T
+    meta: T
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/types.ts:60]: This shouldn't be T.  tx and meta are not the same type.

**Review by JST5000 — APPROVED** [review]
LGTM other than Caleb's comment

**Review by pdp2121 — APPROVED** [review]
LGTM

**Review by ckniffen — APPROVED** [review]

**Pull request merged**